sparqlbuilder/src/main/java/org/eclipse/rdf4j/sparqlbuilder/constraint/SparqlFunction.java

```diff
@@ -54,6 +54,7 @@
 	SHA256("SHA256"),
 	SHA384("SHA384"),
 	SHA512("SHA512"),
+    STR("STR"),
```

Comment:
Minor thing but can you please fix the indentation? We standardize on tabs-only.

Reply:
Done